server/controllers/finance/reports/invoices/index.js

```diff
@@ -162,9 +166,16 @@ function receipt(req, res, next) {
         invoiceResponse.exchangedTotal = _.round(invoiceResponse.cost * invoiceResponse.exchange);
       }
 
+      // render the excel worksheet
+      if (renderer === 'xlsx') {
+        return xlsx.render(invoiceResponse);
```

Comment:
Can you put the `xlsx` into the `receiptReport` renderer?  That way it can all use the same interface?  I think that already controls json/html/pdf versions.  It would make sense to put it there.

Reply:
okay, but we have already an old xlsx render, I'm trying to rewrire it

Reply:
or we can just give this one another name such as xlsx2

Reply:
Aha, good thought.  Hmm...

For now, let's give it another name like xlsx2.  That way, we can get this PR merged and then gradually combine them.